app/styles/components/examples/theming/9_buttonConfigurator.css

```
@@ -0,0 +1,16 @@
+.appearanceExamples__select-group {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(12rem, 1fr));
+    gap: var(--space-4);
+
+    justify-content: center;
+    align-items: start;
+    place-items: center;
```

Biome (2.1.2)

[error] Unexpected shorthand property place-items after align-items



(lint/suspicious/noShorthandPropertyOverrides)